frontend/src/routes/[[lang=locale]]/candidate/(protected)/questions/summary/+page.svelte

```diff
@@ -4,15 +4,16 @@
   import {BasicPage} from '$lib/templates/basicPage';
   import {Expander} from '$lib/components/expander';
   import LikertResponseButtons from '$lib/components/questions/LikertResponseButtons.svelte';
-  import {answerContext} from '$lib/utils/answerStore';
   import Icon from '$lib/components/icon/Icon.svelte';
   import {t} from '$lib/i18n';
   import {getRoute, Route} from '$lib/utils/navigation';
   import {translate} from '$lib/i18n/utils/translate';
   import QuestionOpenAnswer from '$lib/components/questions/QuestionOpenAnswer.svelte';
+  import {getContext} from 'svelte';
+  import type {CandidateContext} from '$lib/utils/candidateStore';
 
-  const store = answerContext.answers;
-  $: answerStore = $store;
+  const {answers: answersStore} = getContext<CandidateContext>('candidate');
+  $: answers = $answersStore;
```

Comment:
Maybe `answers` should be named as `answersStore` in `CandidateContext` because then there would be no need to rename it here and I think it would be clearer (and same thing with renaming `user` as `userStore`)